src/platforms/common/performance/index.mdx

```diff
@@ -51,6 +51,12 @@ Automatic instrumentation for monitoring the performance of your application is
 
 </PlatformSection>
 
+<PlatformSection supported={["javascript.nextjs", "node", "python", "ruby", "java"]} notSupported={["javascript", "dart", "flutter", "react-native", "android", "apple", "dotnet", "javascript.cordova", "javascript.lectron", "go", "unity", "rust", "native", "php", "native.breakpad", "native.crashpad", "native.minidumps", "unreal", "native.wasm"]}>
```

Comment:
copied (not)supported from OTEL page, @tonyo @cleptric need to update this for go once https://github.com/getsentry/sentry-docs/pull/6208 is merged